python/py-absl/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           python 1.0
 
 name                py-absl
-version             0.8.1
+version             0.10.0
```

Comment:
considering updating to the latest version, currently 0.11.0 (I know it was the latest when you submitted this PR...)

Reply:
Thanks done.